src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
```

Comment:
The 'Lottery' class should have an access modifier. It's recommended to specify 'public' explicitly if you intend for this class to be used outside its package.